.github/scripts/spellcheck.words

```diff
@@ -71,6 +71,7 @@ buildbot
 buildconf
 Caddy
 calloc
+cancelling
```

Comment:
This is not how it is spelled!

Reply:
Cancelling, travelling, modelling, ... that's how I spell them. Yet another difference between British and American English. https://english.stackexchange.com/questions/338/when-is-l-doubled